src/components/activity/ActivityDirectiveChangelog.svelte

```diff
@@ -192,18 +192,11 @@
     const { id: activityId, plan_id: planId } = activityDirective;
     activityRevisions = await effects.getActivityDirectiveChangelog(planId, activityId, user);
 
-    // Get default set of effective arguments
-    const effectiveDefaultArguments = await effects.getEffectiveActivityArguments(
-      modelId,
-      activityType?.name || '',
-      {},
-      user,
-    );
-
     // Get effective arguments for all revisions by coalescing defaults with args supplied in revision
-    effectiveRevisionArguments = activityRevisions.map(revision => {
+    const defaultArguments = $activityArgumentDefaultsMap[activityType?.name || ''] ?? null;
+    defaultArguments.effectiveRevisionArguments = activityRevisions.map(revision => {
```

Comment:
Should it be setting an `effectiveRevisionArguments` on the `defaultArguments` object here?

Reply:
Whoops nope, not sure how I ended up with that, good catch. Should just be assigning `effectiveRevisionArguments` to the right side I believe.

Reply:
Fixed